src/app/logic/calculatePoints.js

```diff
@@ -0,0 +1,34 @@
+// eslint-disable-next-line import/extensions
+import { DIFFICULTY_LEVELS } from '../data/consts.js';
+
+export function calculatePoints(
+  correctAnswersScore,
+  isAnswerCorrect,
+  difficultyLevelsProperty,
+) {
+  if (
+    DIFFICULTY_LEVELS[difficultyLevelsProperty].calculatingPoints ===
+    DIFFICULTY_LEVELS.medium.calculatingPoints
+  ) {
+    let newCorrectAnswersScore = correctAnswersScore;
+    if (isAnswerCorrect) {
+      newCorrectAnswersScore = correctAnswersScore + 1;
+    }
+    return newCorrectAnswersScore;
+  }
+  if (
+    DIFFICULTY_LEVELS[difficultyLevelsProperty].calculatingPoints ===
+    DIFFICULTY_LEVELS.hard.calculatingPoints
+  ) {
+    let newCorrectAnswersScore = correctAnswersScore;
+    if (isAnswerCorrect) {
+      newCorrectAnswersScore += 1;
+    } else {
+      newCorrectAnswersScore -= 1;
+    }
+    return newCorrectAnswersScore;
+  }
+  throw new Error('wrong difficulty level property or calculatingPoints value');
```

Comment:
Perhaps you could take the early return approach, which would exit the function before the logic. For example:
`const isMediumMode = diffcultyLevelsProperty === 'medium';
const isHardMode = diffcultyLevelsProperty === 'medium';
if (!isMedium || !isHard) throw...
if (isMedium) {dostg}
if (isHard) {dostg}
`

Reply:
If you want to avoid typing string like "medium" perhaps you could add name key to the difficulties levels object

Reply:
changed! that check is firstly in function. And I replaced strings by indexes of array with DIFFICULTY_LEVELS object Properties (get by Object.getOwnPropertyNames() function).
And later I used boolian const (isMediumMode, isHardMode) instead of that long comparisons